pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1128,6 +1128,9 @@ func TestDryRunPreemption(t *testing.T) {
 				pdbLister: getPDBLister(informerFactory),
 				args:      *tt.args,
 			}
+			diagnosis := framework.Diagnosis{
```

Comment:
We can remove this and use `&framework.Diagnosis{}` in the rest places of this file.